Packs/Core/ReleaseNotes/1_3_35.md

```diff
@@ -0,0 +1,12 @@
+
+#### Layout Rules
+
+##### New: Default XDR Alerts Layout Rule
+
+- New XDR alert default layout rule (Available from Cortex XSOAR 6.10.0).
+
+#### Layouts
+
+##### New: Default XDR Alerts
+
+- New default layout for XDR alerts (Available from Cortex XSOAR 6.10.0).
```

Comment:
Let's remove the (Available from Cortex XSOAR 6.10.0) from both layout +layout trigger